### ISSUE TITLE
Capture the K8s node ready status condition

### ISSUE THREAD

**Issue by h0tbird** [newrelic/nri-kubernetes#55]
## Feature Description
As an engineer responsible for the health of the Kubernetes clusters, I need to make sure the cluster worker nodes are healthy so that they can schedule containers. If too many are NotReady it is a strong indication that the cluster is not healthy.

## Additional context

I would like to capture the k8s node status as an additional attribute in the `K8sNodeSample` event type:
<img width="658" alt="Screenshot 2020-12-02 at 15 01 55" src="https://user-images.githubusercontent.com/417440/100997295-66071100-355a-11eb-9cac-4098b2ffbe87.png">

It looks like the `Ready` `status.condition` can be `True` or `False`:
```
k get no ip-10-215-133-9.us-east-2.compute.internal -o json | jq -r '.status.conditions[] | select( .type == "Ready") | .status'
True
```

## Priority
Really Want

**Comment by h0tbird** [comment]
FWIW I was able to get the data from the `kube-state-metrics` using the `nri-prometheus` and now I can set the alerts using `PromQL`:
```
sum(kube_node_status_condition{condition="Ready",status!="true"}) by (clusterName)
```
or the `NRQL` equivalent:
```
SELECT (average(kube_node_status_condition) * cardinality(kube_node_status_condition)) FROM Metric WHERE ((condition = 'Ready') AND NOT ((status = 'true'))) SINCE 60 MINUTES AGO UNTIL NOW FACET clusterName LIMIT 100 TIMESERIES 30000
```

**Comment by invidian** [comment]
IIRC this will be solved as part of #126 and more specifically #138
